### PR TITLE
44 add a splash screen

### PR DESCRIPTION
**Pull request by adviislam10**

### CODE REVIEW

**Review by Logan9312 — APPROVED** [review]
Seems good

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -2,12 +2,6 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
 
-    <uses-permission
-        android:name="android.permission.READ_EXTERNAL_STORAGE"
-        android:maxSdkVersion="32" />
-
```
Comment (Nandan-18) [inline app/src/main/AndroidManifest.xml:8]: these need to be added back

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -16,11 +10,10 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/Theme.ScanPal"
-        android:usesCleartextTraffic="true"
```
Comment (Nandan-18) [inline app/src/main/AndroidManifest.xml:20]: this param needs to be added back

**Review by Nandan-18 — CHANGES_REQUESTED** [review]

**Review by Logan9312 — APPROVED** [review]

**Review by Sundance636 — APPROVED** [review]

**Review by Nandan-18 — APPROVED** [review]
works as expected

**Review by Logan9312 — APPROVED** [review]

**Review by Sundance636 — APPROVED** [review]
Looks good!

**Review by Nandan-18 — APPROVED** [review]

**Review by Nandan-18 — APPROVED** [review]

**Review by Logan9312 — APPROVED** [review]

**Pull request merged**